resume-buddy/src/test/java/com/google/sps/MatchTest.java

```diff
@@ -235,4 +235,30 @@ public void isNotMatchAndStillHaveTime() throws ParseException {
 
     return actualMatches;
   }
+
+  /** Test sending emails when they are matched */
+  @Test
+  public void testSendingEmailWhenMatched() throws ParseException {
+    // Uncomment when needed to test. Otherwise, will send too much emails.
+    // sendingEmailsWhenMatched();
+  }
+
+  private void sendingEmailsWhenMatched() {
+    REVIEWEE_A.setProperty("email", "animachaidze@gmail.com");
```

Comment:
In Shreya's last PR that was merged in 30 minutes ago, she changed all the `REVIEWER_X` entities to `reviewerX`. I'm surprised this PR is not showing any merge conflicts with googleinterns:master. Can you try to merge from googleinterns:master again?

Reply:
See MatchTest.java in googleinterns:master: https://github.com/googleinterns/ResumeBuddy/blob/master/resume-buddy/src/test/java/com/google/sps/MatchTest.java

Reply:
When I merged, it didn't give me merge conflict but had to change it by hand.